http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -513,7 +513,7 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       Some(userInfo.userEmail),
       None,
       None,
-      Some(RuntimeImageType.Azure)
+      Some(Tool.JupyterLab)
```

Comment:
can u add a `TODO` indicating this needs to fixed once we start supporting `RStudio` on Azure or we start supporting GCP in v2 endpoint